bodhi/client/__init__.py

```diff
@@ -84,6 +103,23 @@ def new(user, password, **kwargs):
 
 
 @updates.command()
+@add_options(new_edit_options)
+@click.argument('builds')
```

Comment:
While using this I realized that we probably don't want to restrict the way that users express which update they want to edit to builds. Some updates have a massive number of builds (like Gnome or KDE) and this will make it difficult to express those updates. I think we want to think of a way to accept update id (like ```FEDORA-2017-2d576b8eb4```). If we can think of a way to use the id or the builds and have the client figure it out, that would also be nice. There's a regex in the bindings that tries to do something like this.